frontend/js/src/artist-similarity/src/index.css

```diff
@@ -11,3 +11,7 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
     monospace;
 }
+
+.graph-container { 
+  height: 92vh;
```

Comment:
We'll probably have to revisit the general page styles once we integrate this page into the website proper.
This arbitrary height for example will probably go.

Reply:
Sure I will keep these things in mind and thanks for the review